src/openforms/typing.py

```diff
@@ -7,3 +10,8 @@
 JSONObject = Dict[str, JSONValue]
 
 DataMapping = Dict[str, Any]  # key: value pair
+
+
+class RequestHandler(Protocol):
+    def __call__(self, request: HttpRequest) -> HttpResponseBase:  # pragma: no cover
+        ...
```

Comment:
Didn't know this worked! Any reason to use this over `pass`?

Reply:
it's convention to use this for typing/overloading :-) see also https://python.land/python-ellipsis